### PR TITLE
Fixed nil pointer dereference in config if a config file lacks "signing"

### PR DESCRIPTION
**Pull request by andrewbuss**
Added regression test which fails without the change

cosmetic: gofmt also fixed some formatting issues

### CODE REVIEW

**Comment by andrewbuss** [comment]
Woops - @j-delaney already fixed this!

File: config/config.go [hunk 1]
```diff
@@ -86,14 +86,15 @@ type SigningProfile struct {
 	CSRWhitelist                *CSRWhitelist
 	NameWhitelist               *regexp.Regexp
 	ClientProvidesSerialNumbers bool
+	CTLog                       string
```
Comment (grittygrease) [inline config/config.go]: This should be saved for a separate commit.
